package.json

```diff
@@ -13,9 +13,9 @@
     "react": "18.1.0",
     "react-dom": "18.1.0",
     "react-resize-detector": "^7.1.2",
-    "react-router-dom": "6.3.0",
+    "react-router-dom": "^5.3.3",
```

Comment:
Is there a reason you want to pin the version back down to 5 instead of 6?

Reply:
Yes, but it's not permanent.

The `activeClassName` prop seems not available. Instead we need to set a new `active` prop to a boolean (and then set our own active class name dependent on that new boolean instead of the old string prop).

To get back up to 6, I think we need to determine how to get a boolean value to pass to new `active` prop (and then use that to dynamically set `className`).

Reply:
> Related to the older versions of react router Dom, etc, let's stick with what is compatible with cep v2 — jchuahtacc [internal comm.](https://tacc-team.slack.com/archives/C02GF2X2AS3/p1655820645562929?thread_ts=1655757404.634289&cid=C02GF2X2AS3)